README.md

```diff
@@ -4,15 +4,41 @@
 Repository of [Cyclus](https://fuelcycle.org/) archetypes to couple Cyclus with [OpenMC](https://docs.openmc.org/en/develop/pythonapi/generated/openmc.run.html)
 
 ## Installation 
-To install this archetype library run ``pip install .``. 
-To run tests: ``pytest`` from the main directory.
+
```

Comment:
```suggestion
Ensure you install Cyclus and OpenMC, and their dependencies, before installing OpenMCyclus.
```

Reply:
In the writing checklist, we say never to have empty sections.

Reply:
Those lines of code weren't supposed to be deleted. Weird.